README.md

```diff
@@ -53,6 +53,26 @@ The code can be re-generated with
     $ git apply scripts/thrift-gen.patch
 ```
 
+### Updating Agent host and port
+
+The default agent host and port is `127.0.0.1:6831`, you can use a different host/port by updating your reporter configuration.
+
+YAML configuration:
+
+```yml
+repoter:
+  localAgentHostPort: my-jaeger-agent.local # or another IP:Port 10.120.1.101:6831
```

Comment:
`my-jaeger-agent.local` looks very confusing to me, what does it mean? Shouldn't there be a port number?